lib/Providers/Qr/EndroidQrCodeProvider.php

```diff
@@ -34,7 +39,7 @@ public function getQRCodeImage($qrtext, $size)
         $qrCode->setBackgroundColor($this->bgcolor);
         $qrCode->setForegroundColor($this->color);
 
-        return $qrCode->writeString();
+        return $qrCode;
```

Comment:
So child classes can tweak the object without messing with how the QRCode image is returned.